src/inverseRobot.js

```diff
@@ -7,7 +7,20 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const valueOfRobot = Object.values(robot);
+  const repairedRobot = {};
+
+  for (let i = 0; i <= valueOfRobot.length; i++) {
+    if (valueOfRobot.indexOf(valueOfRobot[i], i + 1) !== -1) {
+      return null;
+    }
+  }
+
+  for (const key in robot) {
+    repairedRobot[robot[key]] = key;
+  }
```

Comment:
Try to simplify your solution. We can use only one loop for the solution
P.S. Take a look at checklist, we can use `hasOwnProperty` for checking if key is already present